tdex/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.8"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_TDEX_CONNECT_IP
+      APP_PORT: $APP_TDEX_CONNECT_PORT
+  
+  tdexd:
+    container_name: "tdexd"
```

Comment:
We can drop this, and rely on docker-compose to name the container automatically for us.